test/framework/universal_app.go

```diff
@@ -179,6 +179,7 @@ networking:
     redirectPortInbound: %s
     redirectPortInboundV6: %s
     redirectPortOutbound: %s
+    reachableServices: [%s]
 `
```

Comment:
This could really be replaced by by an actual Go object this string templating gets hairy :)

Reply:
What I would really, really, really, really, really, really, really, really like to do is extract service to a deployment. I did that with testserver on Kubernetes. With Universal it's more complicated because of ssh app "framework".